src/firebase/firebase.js

```diff
@@ -2,11 +2,12 @@ import firebase from "gatsby-plugin-firebase"
 
 import { signUpComplete, getCurrentAddress} from "../utils/utils"
 
-let auth, firestore
+let auth, firestore, database
```

Comment:
Codacy found an issue: ['database' is assigned a value but never used.](https://app.codacy.com/gh/treetoplearning/heartwood/pullRequest?prid=5919912)